### PR TITLE
Avoid Attempted to call an undefined method named "getName" with tests

### PR DESCRIPTION
**Pull request by stphnpt**
with tests of toString update

### CODE REVIEW

File: Tests/Admin/BlockAdminTest.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        $admin = new BlockAdmin('sonata.dashboard.admin.block', 'DashboardBundle\Entity\BaseBlock', 'SonataDashboardBundle:BlockAdmin');
+
+        $s = new DummyGetName();
+        $this->assertSame('GetName', $admin->toString($s));
+
+        $s = new DummyGetNameNull();
+        $this->assertSame('GetNameNull', $admin->toString($s));
+
+        $s = new DummyNoGetName();
+        $this->assertSame('NoGetName', $admin->toString($s));
```
Comment (greg0ire) [inline Tests/Admin/BlockAdminTest.php]: Please use a data provider. Some will probably also ask you to move dummy classes to a fixtures namespace (what do you think @sonata-project/contributors ? ).

**Comment by greg0ire** [comment]
Also @stphnpt , there was no need to close #36 : you can just force push new commits to the same branch, and your PR will update.

File: Tests/Admin/BlockAdminTest.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+use Sonata\DashboardBundle\Admin\BlockAdmin;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooGetName;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooGetNameNull;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooNoGetName;
+
+class BlockAdminTest extends \PHPUnit_Framework_TestCase
+{
+    public function testToString()
+    {
+        $admin = new BlockAdmin('sonata.dashboard.admin.block', 'DashboardBundle\Entity\BaseBlock', 'SonataDashboardBundle:BlockAdmin');
```
Comment (OskarStark) [inline Tests/Admin/BlockAdminTest.php]: please line break this

File: Tests/Admin/BlockAdminTest.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ */
+
+namespace Sonata\DashboardBundle\Tests\Admin;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\DashboardBundle\Admin\BlockAdmin;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooGetName;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooGetNameNull;
+use Sonata\DashboardBundle\Tests\Fixtures\Entity\FooNoGetName;
+
```
Comment (OskarStark) [inline Tests/Admin/BlockAdminTest.php]: please add an `@author` tag

**Comment by OskarStark** [comment]
LGTM 👍

**Comment by OskarStark** [comment]
@Soullivaneuh @greg0ire do we have no Style CI check here? Oo

**Comment by Soullivaneuh** [comment]
@OskarStark This project is not already on dev-kit.

Waiting this: https://github.com/sonata-project/SonataDashboardBundle/pull/31

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**